packages/core/src/cart/buy-now-cart-request-body.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * An object that contains the information required for creating 'Buy now' cart.
+ */
+export default interface BuyNowCartRequestBody {
+    source: 'BUY_NOW';
+    lineItems: Array<{
+        productId: number;
+        quantity: number;
+        optionSelections?: {
+            optionId: number;
+            optionValue: number | string;
+        };
```

Comment:
Can we move this to a cart post line item?

Then it can be LineItem[].

Reply:
Great point. Thanks!